docs/apis-tools/administration-sm-api/specifications/administration-api-self-managed.info.mdx

```diff
@@ -13,6 +13,10 @@ import SchemaTabs from "@theme/SchemaTabs";
 import TabItem from "@theme/TabItem";
 import Export from "@theme/ApiExplorer/Export";
 
+<span className={"theme-doc-version-badge badge badge--secondary"}>
+  Version: 1.0.0
+</span>
+
```

Comment:
Oops, this needs to be fixed, it results in a duplicate badge showing. This is an indication that I have an incorrect path in the generation strategy logic for this API. Fixing....

Reply:
fixed in latest commit